remix-gql-tailwind/server.ts

```diff
@@ -0,0 +1,11 @@
+import { createRequestHandler } from "@remix-run/architect";
+import * as build from "@remix-run/dev/server-build";
+
+// if (process.env.NODE_ENV !== "production") {
+//   require("./mocks");
+// }
```

Comment:
is there a reason this is just commented out for now vs just removing it if it's not needed?

Reply:
It's currently commented out as we don't have `./mocks` fleshed out and I don't want to miss it. Let me add a TODO for clarity.